### ISSUE TITLE
Polyfill slows down readability.js

### ISSUE THREAD

**Issue by MrOrz** [cofacts/url-resolver#4]
Reproduce:

1. Start server with launch param `devtools: true`
2. Use the following GraphQL:

```
{
  resolvedUrls(urls: ["http://blog.renren.com/share/300233063/14628038408"]) {
    url
  }
}
```

3. The GraphQL never returns.
4. To debug, you can pause script in devtools in "scripts" panel 

You will find that the script always pauses at `String.prototype.trim` polyfill, and the script never ends.

The page is using a slow polyfill (implements `trim()` using regex) while readability.js relies heavily on `trim()`, causing the page to hang.

One possible solution would be that once the page is loaded, remove all script tags, take its HTML to a fresh page, render it there and then invoke readability. This should fix other cases like:
- http://blog.udn.com/watercmd/1066441 - `forEach` polyfill causing page to fail
- https://pension.president.gov.tw/cp.aspx?n=0710ED8C9356A871 - `window.URL` overridden with a string

**Comment by MrOrz** [comment]
page.setContent would be useful when starting a new page with cleansed HTML:

https://github.com/GoogleChrome/puppeteer/blob/v1.8.0/docs/api.md#pagesetcontenthtml

We even don't need to manipulate DOM to disable JavaScript:

https://github.com/GoogleChrome/puppeteer/blob/v1.8.0/docs/api.md#pagesetjavascriptenabledenabled

**Comment by MrOrz** [comment]
Fixed by 9ee9ef944003f2661d322f01d334f5374738e6ba
with an exception:
https://pension.president.gov.tw/cp.aspx?n=0710ED8C9356A871 - cannot be fixed because 
URL must be resolved under original page
